pwndbg/lib/memory.py

```diff
@@ -94,7 +94,7 @@ def is_stack(self) -> bool:
 
     @property
     def is_memory_mapped_file(self) -> bool:
-        return len(self.objfile) != 0 and self.objfile[0] != "[" and self.objfile != "<pt>"
+        return len(self.objfile) != 0 and self.objfile[0] != "["
```

Comment:
That last check here is obsolete after we rename from `<pt>` to `[pt_<addr>]` :)